### ISSUE TITLE
Add "default" property for optional parameters automatically

### ISSUE THREAD

**Issue by guangyang** [domaindrivendev/Swashbuckle#369]
Right now I need to use some IOperationFilter to do this like:

```csharp
public class ParameterDefaultValueFilter : IOperationFilter
{
    public void Apply(Operation operation, SchemaRegistry schemaRegistry, ApiDescription apiDescription)
    {
        var parameterDescriptors = apiDescription.ActionDescriptor.GetParameters();
        for (int i = 0; i < parameterDescriptors.Count; i++)
        {
            var pd = parameterDescriptors[i];
            if (pd.IsOptional && !(pd.ParameterBinderAttribute is FromBodyAttribute))
            {
                operation.parameters[i].@default = pd.DefaultValue;
            }
        }
    }
}
```

According to Swagger 2.0, default value on a parameter is supported as long as it's not from body. Could we add this support to Swashbuckle directly?

**Comment by domaindrivendev** [comment]
Should be resolved in latest release - 5.2.0

**Comment by tynor88** [comment]
Is there some way you need to activate this? I have tried upgrading to 5.2.0, but my default params doesn't show without using the above code?
